### DISCUSSION TITLE
Organising packages in a universe

### DISCUSSION THREAD

**Discussion by drmowinckels** [r-universe-org/help#100] [Ideas]
Hi.

During a community call in the spring, I suggested finding a way to organise the packages in a universe in some way. 
There could be many types of packages in a universe, and having a way of organising these in the "Build" tab (even maybe in the "Packages" tab, though that might be ott). 

For instance, say you have a collection of standard packages, with mainly functions that you'd like to have under a category "packages". Another set which are really just data packages, that could go under the category "data". And yet another set of tutorials or research compendia. Currently, all packages look equal, and it can be hard to tell what is what from the universe. But if the universe json could include a field for categorising a universe's packages in some way that would sort them or something, that would be a very tidy addition.

**Comment by jeroen** [comment]
Thanks for that suggestion. Perhaps we can combine this with e.g. https://github.com/r-universe-org/bugs/issues/68 to allow for customizing the appearance of a universe.

**Reply by drmowinckels** [reply]
Thats an excellent idea! I really like the suggestion about a landing page through a README.

**Comment by ThierryO** [comment]
I like this idea. Maybe allow adding categories to the `packages.json`. And add a `categories.json` to the repo. I took categories from https://ropensci.org/packages/

We could have a generic `categories.json` that groups packages at https://r-universe.dev. The `categories.json` in `github.com/organisation/universe` would only affect https://organisation.r-universe.dev

_packages.json_
``` 
    {
        "package": "git2rdata",
        "url": "https://github.com/ropensci/git2rdata",
        "category": ["data_publication", "data_extraction"]
    }
```

_categories.json_
```
    {
        "category": "data_publication",
        "description:" "Document and Release Your Data"
    },
    {
        "category": "data_extraction",
        "description:" "Convert and Munge Data"
    }
```
